www/source/javascripts/app.js

```diff
@@ -4,6 +4,8 @@
 
 /* To load a config file (particles.json) you need to host this demo (MAMP/WAMP/local)... */
 
+var animate = !matchMedia('(prefers-reduced-motion)').matches;
+
```

Comment:
I would prefer if we could use `window.matchMedia('(prefers-reduced-motion)').matches`